setup.py

```diff
@@ -26,6 +26,7 @@
     'recommonmark>=0.4.0',
     'Sphinx>=1.3.5',
     'sphinxcontrib-napoleon>=0.4.4',
+    'sphinx-rtd-theme>=0.1.9',
```

Comment:
I think we could fully really on `setup.py`, perhaps? See https://github.com/bigchaindb/bigchaindb/blob/develop/setup.py#L25